### PR TITLE
Replace Pokégear with Trainer Gear in relevant labels

### PR DESCRIPTION
**Pull request by SatoMew**
The early implementation of the Pokémon Gear (ポケモンギア) or Pokégear (ポケギア) was called Trainer Gear (トレーナーギア) in the demo so this commit is to help maintaining consistency with the in-game terminology.

### CODE REVIEW

**Comment by Rangi42** [comment]
@SatoMew Do you want to resolve the conflicts with this PR so it can be merged?

**Comment by SatoMew** [comment]
Please verify that they have been resolved correctly.

File: Makefile [hunk 1]
```diff
@@ -98,9 +98,7 @@ $(BUILD)/gfx/sgb/corrupted_b1e3.2bpp: tools/gfx += --trim-whitespace
 $(BUILD)/gfx/sgb/sgb_border_silver.2bpp: tools/gfx += --trim-whitespace
 
 $(BUILD)/gfx/trainer_card/leaders.2bpp: tools/gfx += --trim-whitespace
-
-$(BUILD)/gfx/pokegear/town_map.2bpp: tools/gfx += --trim-trailing
-
+$(BUILD)/gfx/trainer_gear/town_map.2bpp: tools/gfx += --trim-trailing
```
Comment (Rangi42) [inline Makefile:101]: It's nitpicking, but this could keep the blank lines separating it from unrelated gfx/\*/ subdirectory lines.

Comment (SatoMew) [reply Makefile:102]: Fixed.

**Pull request merged**